scripts/install.sh

```diff
@@ -55,8 +55,8 @@ cat << EOF
 OPTIONS:
    -h      Show help.
    -d      Directory where kubent will be installed. Default is /usr/local/bin
-   -a      Architecture to install (x86_64 only atm.). Default is to auto-detect.
-   -o      OS (linux, macos). Default is to auto-detect.
+   -a      Architecture to install (amd64 or arm64). Default is to auto-detect.
+   -o      OS (linux, macos, windows (windows amd64 only). Default is to auto-detect.
```

Comment:
```suggestion
   -o      OS (linux, macos, windows [amd64 only]). Default is to auto-detect.
```

Missing closing parenthesis, also use `[]` for the inner ones (although I would maybe leave out the info about arch for windows altogether for clarity).